src/browser/main.coffee

```diff
@@ -56,11 +56,24 @@ handleStartupEventWithSquirrel = ->
 setupCrashReporter = ->
   crashReporter.start(productName: 'Atom', companyName: 'GitHub')
 
-setupAtomHome = ->
+isPortableInstall = (args) ->
+  return false unless process.platform is 'win32'
+  return false unless (process and process.type)
```

Comment:
I don't think you need the `process and` check here, since the line before already calls `process.platform` without checking `process`.